tests/test_cli.py

```diff
@@ -1,19 +1,81 @@
+import sys
+
 from click.testing import CliRunner
+from _pytest.pytester import LineMatcher
+
+from hookman import __main__
 
-from hookman import cli
+
+def test_help(datadir):
+    runner = CliRunner()
+    result = runner.invoke(__main__.cli)
+    assert result.exit_code == 0, result.output
+    matcher = LineMatcher(result.output.splitlines())
```

Comment:
Wait, Wat! Did you know the existence of this LineMatch all this time and never told anyone about this?

I'm going to change all my tests right now!

Reply:
Heh 😁